cf-check/dump.c

```diff
@@ -346,6 +346,31 @@ int dump_db(const char *file, const dump_mode mode)
     return 0;
 }
 
+static bool matches_option(
```

Comment:
Looks like a good candidate for a function in *libutils*.

Reply:
Yep, I'll do it in a follow-up PR.